doc/tutorial_ti_cc3200.md

```diff
@@ -96,42 +98,45 @@ WolfSSL is used to create secure TLS connections.  There is a version of WolfSSL
+The wolfSSL supports TI-RTOS builds. Below we've provided the needed `{..}/wolfssl/tirtos/products.mak` variable settings for Windows and MacOS to make it easier to get going:
 
-Example tirtos/products.mak variable settings for Windows and MacOS:
+_Alternatively you can follow the steps written on [Using wolfSSL with TI-RTOS](http://processors.wiki.ti.com/index.php/Using_wolfSSL_with_TI-RTOS) to generate wolfSSL static library for CC3200._
 
-- Windows:
+#### Windows:
 
-        XDC_INSTALL_DIR        =c:/ti/xdctools_3_32_01_22_core
-        BIOS_INSTALL_DIR       =c:/ti/tirex-content/tirtos_cc32xx_2_16_00_08/products/bios_6_45_01_29
-        NDK_INSTALL_DIR        =
-        TIVAWARE_INSTALL_DIR   =
+    XDC_INSTALL_DIR        =c:/ti/xdctools_3_32_01_22_core
+    BIOS_INSTALL_DIR       =c:/ti/tirex-content/tirtos_cc32xx_2_16_00_08/products/bios_6_45_01_29
+    NDK_INSTALL_DIR        =
+    TIVAWARE_INSTALL_DIR   =
 
-        export XDCTOOLS_JAVA_HOME=c:/Program Files (x86)/Java/jre1.8.0_51
+    export XDCTOOLS_JAVA_HOME=c:/Program Files (x86)/Java/jre1.8.0_51
 
-        ti.targets.arm.elf.M4F =c:/ti/ccsv6/tools/compiler/arm_15.12.3.LTS
-        iar.targets.arm.M4F    =
-        gnu.targets.arm.M4F    =
+    ti.targets.arm.elf.M4F =c:/ti/ccsv6/tools/compiler/arm_15.12.3.LTS
+    iar.targets.arm.M4F    =
+    gnu.targets.arm.M4F    =
 
-- MacOS:
+#### MacOS:
+
+    XDC_INSTALL_DIR        =/Applications/ti/xdctools_3_32_01_22_core
+    BIOS_INSTALL_DIR       =$(HOME)/ti/tirex-content/tirtos_cc32xx_2_16_00_08/products/bios_6_45_01_29
+    NDK_INSTALL_DIR        =
+    TIVAWARE_INSTALL_DIR   =
 
-        XDC_INSTALL_DIR        =/Applications/ti/xdctools_3_32_01_22_core
-        BIOS_INSTALL_DIR       =/Users/atigyi/ti/tirex-content/tirtos_cc32xx_2_16_00_08/products/bios_6_45_01_29
-        NDK_INSTALL_DIR        =
-        TIVAWARE_INSTALL_DIR   =
+    export XDCTOOLS_JAVA_HOME=/Applications/ti/ccsv6/eclipse/jre/Contents/Home
 
-        export XDCTOOLS_JAVA_HOME=/Applications/ti/ccsv6/eclipse/jre/Contents/Home
+    ti.targets.arm.elf.M4F =/Applications/ti/ccsv6/tools/compiler/arm_15.12.3.LTS
+    iar.targets.arm.M4F    =
+    gnu.targets.arm.M4F    =
 
-        ti.targets.arm.elf.M4F =/Applications/ti/ccsv6/tools/compiler/arm_15.12.3.LTS
-        iar.targets.arm.M4F    =
-        gnu.targets.arm.M4F    =
+**Important Note** - Depending on the version of the packages installed, the folder of `BIOS_INSTALL_DIR` may be different. Please check inside the `~/ti/tirex-content` folder to ensure the variable references the correct folder.
 
-Further wolfSSL build customizations:
+#### Further wolfSSL build customizations:
 
-- In file wolfssl/wolfcrypt/settings.h add a new platform macro WOLFSSL_NOOS_XIVELY with content:
+- In the file `{..}/wolfssl/wolfssl/wolfcrypt/settings.h` add a new platform macro `WOLFSSL_NOOS_XIVELY` with the following content. This will configure the wolfSSL features needed for connecting to the Xively service.
```

Comment:
Please add this or the more english / context friendly version of it:
"Position of this macro matters. Please put the new section just before the line
#ifdef WOLFSSL_TIRTOS"